source/dotnet/domain-tests/Fixtures/AuthorizedClientFixtureOutput.cs

```diff
@@ -110,6 +110,11 @@ private async Task<bool> WaitForCalculationToComplete(Guid calculationId)
                 defaultDelay);
             stopwatch.Stop();
             Console.WriteLine($"LOOK AT ME: Calculation took {stopwatch.Elapsed} to complete for calculationId {calculationId}");
+            if (stopwatch.Elapsed >= TimeSpan.FromMinutes(15))
+            {
+                Console.WriteLine($"LOOK AT ME: Calculation with calculationId {calculationId} took more than 15 minutes to complete. And failed the test");
```

Comment:
What is it about that "LOOK AT ME" thing? 😄 
How about using Error?
```suggestion
                Console.Error.WriteLine($"Calculation with ID {calculationId} took more than 15 minutes to complete");
```

Reply:
Comment/question applies to all our current similar logging.

Reply:
will do